common/djangoapps/entitlements/api/v1/serializers.py

```diff
@@ -24,3 +29,40 @@ class Meta:
             'mode',
             'order_number'
         )
+
+
+class CourseEntitlementSupportDetailSerializer(serializers.ModelSerializer):
+    """ Serialize the details of a support team interaction with a learner's course entitlement. """
+    support_user = serializers.SlugRelatedField(
+        read_only=True,
+        slug_field='username',
+        default=serializers.CurrentUserDefault()
+    )
+    unenrolled_run = CourseKeyField('unenrolled_run.id')
```

Comment:
Did you remove reason completely on purpose?

Reply:
It's not actually removed from the serializer, it will just use default behavior for the field type. If you look below you'll see it's still included in the fields to be serialized.